single-sign-on/faqs.md

```diff
@@ -5,83 +5,215 @@ title: Single Sign-on FAQs
+
+### Q: How do I handle accounts using Docker Hub as a secondary registry? Do I need a bot account?
+
+You can add a bot account to your IDP and create an access token for it to replace the other credentials.
+
+### Q: Does Docker plan to release SAML just in time provisioning?
+
+Our SSO implementation is already "just in time". Admins don't have to create users’ accounts on Hub, they can just enable it on the IdP and have the users log in via their domain email on Hub.
+
+### Q: Will there be IdP initiated logins? Does Docker plan to support SSO logins outside of Hub and Desktop?
+
+We currently do have any plans to enable IdP initiated logins.
+
+### Q: Build agents - For customers using SSO, do they need to create a bot account to fill a seat within the dockerorg?
+
+Yes, generally bot accounts need to be a seat, similar to a regular end user, having a non-aliased domain email enabled in the IdP and using a seat in Hub.
+
+### Q: Is it possible to connect Docker Hub directly with a Microsoft Azure Active Directory Group?
+
+Yes, Azure AD is supported with SSO for DB, both via a direct integration and via SAML.
+
+## Adding domain and domain verification
+
+### Q: What should I do if I reach the character limits when adding the txt record for my domain?
+
+Yes, you can add sub-domains to your SSO , however all email addresses should also be on that domain. Verify that your DNS provider supports multiple txt fields for the same domain.
+
+### Q: Can the DNS provider configure it once for one-time verification and remove it later OR will it be needed permanently?
+
+They can do it one time to add it to a connection. If they ever change idPs and have to set up SSO again, they will need to verify again.
+
+
+### Q: Is adding Domain required to configure SSO? What domains should I be adding? And how do I add it?
+
+Adding and verifying Domain is required to enable and enforce SSO. Click Add Domain and specify the email domains that are allowed to authenticate via your server. This should include all email domains users will use to access Docker. Public domains are not permitted, such as gmail.com, outlook.com, etc. Also, the email domain should be set as the primary email.
+
+### Q: How can we send a verification e-mail to just the domain reference? Wouldn’t we need a full e-mail address?
+
+
+### Q: What would the full content of the registry.json need to be? Can an example be provided?
+
+### Q: Does the registry.json go on the user’s workstation?
+
+### Q: If users are using their personal email, do they have to convert to using the Org’s domain before they can be invited to join an Org? Is this just a quick change in their Hub account?
+
+No, they do not. Though they can add multiple emails to a dockerid if they choose to. However, that email can only be used once across docker. The other thing to note is that (as of January 2022) SSO will not work for multi domains as a MVP and it will not work for personal emails either.
```

Comment:
```suggestion
No, they do not. Though they can add multiple emails to a Docker ID if they choose to. However, that email can only be used once across Docker. The other thing to note is that (as of January 2022) SSO will not work for multi domains as an MVP and it will not work for personal emails either.
```